run-tests.php

```diff
@@ -2109,6 +2109,12 @@ function run_test($php, $file, $env)
 	if (array_key_exists('INI', $section_text)) {
 		$section_text['INI'] = str_replace('{PWD}', dirname($file), $section_text['INI']);
 		$section_text['INI'] = str_replace('{TMP}', sys_get_temp_dir(), $section_text['INI']);
+		if (PHP_OS_FAMILY === 'Windows') {
```

Comment:
You could use the IS_WINDOWS constant here

Reply:
Ah, very nice! Thanks.